dm/dm/worker/source_worker.go

```diff
@@ -623,25 +623,26 @@ func (w *SourceWorker) OperateSubTask(name string, op pb.TaskOp) error {
 		return terror.ErrWorkerSubTaskNotFound.Generate(name)
 	}
 
+	w.l.Info("OperateSubTask start", zap.Stringer("op", op), zap.String("task", name))
 	var err error
 	switch op {
-	case pb.TaskOp_Stop:
-		w.l.Info("stop sub task", zap.String("task", name))
+	case pb.TaskOp_Delete:
+		w.l.Info("delete subtask", zap.String("task", name))
 		st.Close()
 		w.subTaskHolder.removeSubTask(name)
-	case pb.TaskOp_Pause:
-		w.l.Info("pause sub task", zap.String("task", name))
+	case pb.TaskOp_Pause, pb.TaskOp_Stop:
+		w.l.Info("pause subtask", zap.String("task", name))
 		err = st.Pause()
 	case pb.TaskOp_Resume:
-		w.l.Info("resume sub task", zap.String("task", name))
+		w.l.Info("resume subtask", zap.String("task", name))
 		err = st.Resume(w.getRelayWithoutLock())
 	case pb.TaskOp_AutoResume:
-		w.l.Info("auto_resume sub task", zap.String("task", name))
+		w.l.Info("auto_resume subtask", zap.String("task", name))
```

Comment:
if we unify `pause -> stop`, `resume -> start` later, what should we call this `auto resume`, `auto start`? sounds confusing.